Tests/Logic/ObjectMapping/RKObjectMappingTest.m

```diff
@@ -267,4 +267,27 @@ - (void)testInverseMappingWithNilDestinationKeyPathForAttributeMapping
     expect(operation.destinationObject).to.equal(@{ @"Blake": @{} });
 }
 
+- (void)testRemoveAttributeMappingWithNilDestinationKeyPath
+{
+    // This test fails also if we create directly a mapping from "(something) => (null)"
+    // but the inverseMapping case is a more common use case
+    RKObjectMapping *mapping = [RKObjectMapping mappingForClass:[RKTestUser class]];
+    [mapping addPropertyMapping:[RKAttributeMapping attributeMappingFromKeyPath:nil toKeyPath:@"name"]];
+
+    RKObjectMapping *inverseMapping = [mapping inverseMapping];
+    NSException *exception = nil;
+    @try {
```

Comment:
instead of the try/catch stuff, just assert that the inverse mapping has the properties you expect